> I mean have you tested the preset in the version I proposed?

Here's the results of my attempts. I had to install catch2, before I could configure using 'linux-armv7' (I never had to use catch2 earlier):

```
cmake --preset linux-armv7
# (OK)
cmake --build --preset linux-armv7-release
CMake Error: No such build preset in /Users/jens/Files/Sources/endless-sky: "linux-armv7-release"
cmake --build --preset linux-release
Error: /Users/jens/Files/Sources/endless-sky/build/linux is not a directory
```
Edit: Sorry for the late reply, I was in the middle of trying out your pull-request last time, and I ran out of data on my mobile subscription, so I couldn't do anything - not even post a message on github.

--- a/CMakePresets.json
+++ b/CMakePresets.json
@@ -399,6 +399,18 @@
 			"inherits": "release"
 		},
 		{
+			"name": "linux-armv7-debug",
+			"displayName": "Debug",
+			"configurePreset": "linux-armv7",
+			"inherits": "debug"
+		},
+		{
+			"name": "linux-armv7-release",
+			"displayName": "Release",
+			"configurePreset": "linux-armv7",
+			"inherits": "release"
+		},
+		{
 			"name": "macos-debug",
 			"displayName": "Debug",
 			"configurePreset": "macos",

